src/electionguard/constants.py

```diff
@@ -75,7 +77,7 @@ def get_constants() -> ElectionConstants:
     return option_map.get(option) or STANDARD_CONSTANTS
 
 
-get_large_prime = lambda: get_constants().large_prime
-get_small_prime = lambda: get_constants().small_prime
-get_cofactor = lambda: get_constants().cofactor
-get_generator = lambda: get_constants().generator
+get_large_prime = lambda: int(get_constants().large_prime.value)
```

Comment:
I'm mildly concerned about the performance here. Any alternate version may be adding mpz to comparisons in BigInteger.

Reply:
This could be a bug created for later if we wish and we could remove this.